code/_onclick/click.dm

```diff
@@ -118,7 +118,9 @@
 		RestrainedClickOn(A)
 		return
 
-	if(in_throw_mode && throw_item(A))
+	if(throw_mode)
+		changeNext_move(CLICK_CD_THROW)
+		throw_item(A)
 		return
```

Comment:
Does this change the behaviour of throwing? If I enter throw mode and and click eithout being thrown are things still the same? I remember this change was made because throwing was quite inconsistent before.

Reply:
Adjusted this, check latest commit.